opentreemap/importer/models/trees.py

```diff
@@ -91,8 +91,6 @@ class TreeImportRow(GenericImportRow):
         'geom': fields.trees.POINT,
         'width': fields.trees.PLOT_WIDTH,
         'length': fields.trees.PLOT_LENGTH,
-        # TODO: READONLY restore when implemented
```

Comment:
When searching for `readonly` in the code I still see several references to it. Since readonly is not related to hstore, I would rather not have a partial removal of it mixed up in this PR.